gfx/UltimateCataclysm/pngs_tall_32x64/monsters/mon_zombie_headless_variants/mon_zombie_headless.json

```diff
@@ -1,10 +1,14 @@
 [
-{
+  {
   "id": "mon_zombie_brainless",
     "fg": [
       { "weight": 100, "sprite": "zombie_headless_f" },
       { "weight": 100, "sprite": "zombie_headless_m" }
     ],
   "bg": "shadow_32x64_normal"
-}
+  },
+  {
+    "id": "corpse_mon_zombie_brainless",
+    "fg": "corpse_mon_zombie_brainless",
```

Comment:
```suggestion
    "fg": "corpse_mon_zombie_brainless"
```

Reply:
@MrAsbestos `bg` should not be required now, can you please try removing `bg` and the comma? Yes, it works this way, I just would like to show and confirm that it will work without bg